tests/data/wien2k/core_hole/TiN-corehole.inc

```diff
@@ -0,0 +1,29 @@
+ 4 0.00  0  NUMBER OF ORBITALS (EXCLUDING SPIN), SHIFT, IPRINT
```

Comment:
Did I provide just the scf and inc file (I don't remember now and can§t find the email)? If you have the full directory, commiting all of it to the tests would be preferable.

Reply:
Okay, I added the rest of TiN-corehole files to the directory and renamed it.